extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -478,6 +480,37 @@ describes.realWin('amp-ad-network-doubleclick-impl', realWinConfig, env => {
       });
     });
 
+    describe('data-force-safeframe', () => {
+      const regexp = /(\?|&)fsf=1(&|$)/;
```

Comment:
What about `const fsf_param = ...`?

Reply:
Went with fsfRegexp as its camel-case and reflect regexp